scripts/setup/install_viewer.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+
+set -eux
+
+# Install cbmc-viewer 2.5
+wget https://github.com/awslabs/aws-viewer-for-cbmc/releases/download/viewer-2.5/cbmc_viewer-2.5-py3-none-any.whl \
```

Comment:
Making viewer version a variable might make it easier to update in the future as viewer releases new versions

Reply:
Changed the script so that version is passed as an argument to the script. Is that okay?